features/robots.feature

```diff
@@ -0,0 +1,7 @@
+Feature: Site blocks web spiders
+
+  Scenario: Web spider looks for /robots.txt 
+    Given I am an unauthenticated user
+    When I visit '/robots.txt'
+    Then I see 'User-Agent: *'
```

Comment:
The examples I've seen use `User-agent: *` (lowercase 'a'). Not sure if it's significant.